common/ellipses.js

```diff
@@ -22,39 +22,65 @@
                 };
             }
 
-            if (chaiseConfig.confirmDelete === undefined || chaiseConfig.confirmDelete) {
-                var onError = function (response) {
+            var onError = function (hasConfirm) {
+                return function (response) {
                     scope.$root.showSpinner = false;
                     // if response is string, the modal has been dismissed
-                    if (typeof response !== "string") {
+                    if (!hasConfirm || typeof response !== "string") {
                         ErrorService.handleException(response, true);  // throw exception for dismissible pop- up (error, isDismissible = true)
                     }
                 }
+            }
+
+            var deleteTuple = function (hasConfirmation) {
```

Comment:
The following comments are not very important, but are things that I think can help the readability of code. They are subjective and you might disagree with me about them. In that case you can just ignore the comment.

- It's better to move all the variable declaration top of the function. For example the tuples is defined at the bottom of the function, but used in `deleteTuple`. It's a bit hard to follow where that is coming from. Also the name `tuples` is a bit misleading, you can just pass `[dataForDelete]`.

- I think the `forEach` syntax is easier to read. It also makes it easier since you don't have to repeat a lot. We should try using that as much as we can. If we want to move to ES6 at some point, it makes that transition easier too.
Instead of:
  ```javascript
    for (var k=0; k<fk.colset.columns.length; k++) {
      var col = fk.colset.columns[k];
      ...
    }
   ```
    It can be

    ```javascript
      fk.colset.columns.forEach(function (col) { 
          ...
     });
    ```


Again these are not that important and are somethings that came to my mind by looking at the code. You can change them or leave them the way they are, it's up to you.